src/pks/flow/overland_pressure_pk.cc

```diff
@@ -1191,7 +1191,9 @@ void OverlandPressureFlow::FixBCsForOperator_(const Teuchos::Ptr<State>& S,
             " Aff "<< Aff[f](0,0) << "\n";
         }
       }
+
```

Comment:
Again, I'd suggest using emacs whitespace-cleanup or comparable to remove spaces in empty lines, and there is no need to add blank lines all over the place.  The formatting norms for both Amanzi and ATS are 1 blank line to break up logical breaks in code, and 2 blank lines between function definitions.